tools/azure-sdk-tools/packaging_tools/generate_utils.py

```diff
@@ -126,3 +127,31 @@ def judge_tag_preview(path: str) -> bool:
 
     _LOGGER.info(f'find single api version:{api_version}')
     return 'preview' in api_version
+
+
+def format_samples(sdk_code_path) -> None:
+    generate_sample_path = Path(sdk_code_path + '/generate_sample')
+    if not os.path.exists(generate_sample_path):
+        _LOGGER.info(f'not find generate_sample')
+        return
+
+    try:
+        import black
+    except Exception as e:
+        check_call('pip install black', shell=True)
+        import black
+
+    _BLACK_MODE = black.Mode()
+    _BLACK_MODE.line_length = 120
```

Comment:
```suggestion
```

Reply:
> 

In autorest.python, the line length is changed to 120. just consistent with it.

fyi: https://github.com/Azure/autorest.python/blob/HEAD/autorest/black/__init__.py#L18